frontend/src/pages/AccountManagement/components/Groups.js

```diff
@@ -116,7 +116,7 @@ export default function Groups() {
 
       <div className="margin-bottom-3 maxw-tablet-lg">
         <h3>Created by others (public)</h3>
-        {!groups || !groups.publicGroups.length ? <p className="usa-prose">No one has made any groups in your region public yet</p> : (
+        {!groups || !groups.publicGroups.length ? <p className="usa-prose">No one in your region has created a public group.</p> : (
```

Comment:
Maybe a dumb question but do we think this language might deter anyone from making a group in their region.

For example, I don't want to be the first.